melt_czochralski_Dadzis/multilog_eln.archive.yaml

```diff
@@ -1,27 +1,42 @@
 definitions:
```

Comment:
Is the melt_czochralski.schema.archive.yaml file still used?

Reply:
It's now all included in multilog_eln.archive.yaml

Reply:
Okay, let's keep it like this for now. If we will experience issue then we can always come back to separate schema and data files again. 
Whenever you will upload a file containing a schema, you will have it available in nomad for further experiments you want to record within the same data structure. The idea was to put all the schemas in one folder and separate them from the instances you create out of it. But I believe you will figure out your better routine along the way